Gemfile.lock

```diff
@@ -1038,10 +1038,10 @@ GEM
 
 PLATFORMS
   java
+  ruby
   x64-mingw32
   x86-mingw32
   x86-mswin32
-  x86_64-darwin-22
```

Comment:
Please add this back.

Reply:
Trevor mentioned removing it earlier https://dsva.slack.com/archives/CBU0KDSB1/p1709065471337669?thread_ts=1707152032.413019&cid=CBU0KDSB1

Reply:
Perfect! Just didn't know why this PR was removing the platform